configs/stable_diffusion/stable-diffusion_ddim_denoisingunet.py

```diff
@@ -47,12 +48,15 @@
     set_alpha_to_one=False,
     clip_sample=False)
 
-init_cfg = dict(type='Pretrained', pretrained_model_path='')
-
 model = dict(
     type='StableDiffusion',
-    diffusion_scheduler=diffusion_scheduler,
     unet=unet,
     vae=vae,
-    init_cfg=init_cfg,
-)
+    text_encoder=dict(
+        type='ClipWrapper',
+        clip_type='huggingface',
+        pretrained_model_name_or_path='runwayml/stable-diffusion-v1-5',
+        subfolder='text_encoder'),
+    tokenizer='runwayml/stable-diffusion-v1-5',
```

Comment:
why tokenizer is directly from runwayml but text_encoder needs a clipwrapper?

Reply:
This is a good idea, I will support a wrapper for the tokenizer in textual inversion.